### ISSUE TITLE
GitHub Organizational Accounts

### ISSUE THREAD

**Issue by alanzychflynn** [dependencyci/support#45]
How can I get dependencyci to see my organizational repos?

**Comment by andrew** [comment]
You might need to go to this link: https://dependencyci.com/enable_private and then sync your account

**Comment by alanzychflynn** [comment]
That works fine for me but it doesn't appear to allow the other members of my organization to have access to the repos in my trial account.

**Comment by andrew** [comment]
@alanzychflynn have they logged in and used that link too?

**Comment by alanzychflynn** [comment]
Yes, but if I've started an Organization trial I'm not sure why each individual in my organization would have to start their own trial.

If I've signed up for the organization shouldn't everyone in my organization be counted as a collaborator and rolled into my trial?

Thanks,

**Comment by andrew** [comment]
They shouldn't need to start a trial but they will need to enable access to private repos do dependencyci can check their access level with the GitHub API

**Comment by stahnma** [comment]
We're having this problem as well. I can't use this service if collaborators on a repo can't view the details of the failures on a PR.
